.gitlab-ci.yml

```diff
@@ -0,0 +1,21 @@
+stages:
+  - build
+  - post_build
```

Comment:
I have some concern that subsequent stages only execute if prior stages succeed... If `build` fails, does `post_build` still execute?